### ISSUE TITLE
Support setting alternative text in existing content

### ISSUE THREAD

**Issue by JakePT** [JakePT/automatic-alternative-text#9]
Investigate the possibility of updating alt text for images in _content_.

Updating alt text in the media library will update alt text for any images used in correctly developed templates, or as featured images, but won't help for images added to content using the Classic or Gutenberg editor.

This feature would need to scan post/page content HTML to find images missing alt text, send the URL to the API, and then update the HTML. Need to figure out how to handle large numbers of posts, and whether there'll be an issue with breaking Image and Gallery blocks if the HTML is updated directly.
